rest/model/version.go

```diff
@@ -25,7 +25,7 @@ type APIVersion struct {
 	Branch             *string        `json:"branch"`
 	Parameters         []APIParameter `json:"parameters"`
 	BuildVariantStatus []buildDetail  `json:"build_variants_status"`
-	Builds             []APIBuild     `json:"builds"`
+	Builds             []APIBuild     `json:"builds,omitempty"`
```

Comment:
Is there a specific reason you're changing this? It's a public interface change which could break other people's scripts

Reply:
I'm the one who added builds in the last PR for this route; I was thinking it would be confusing to see it in other routes.